### DISCUSSION TITLE
Pull request question

### DISCUSSION THREAD

**Discussion by Masteroshi430** [zwave-js/zwave-js#5084] [General] [answered]
Hi, I did a pull request to update the QUBINO SMART PLUG 16A (0x0159) config file but it says "This commit does not belong to any branch on this repository" and it doesn't appear in the pull request list and I don't know what to do.
Here it is:
https://github.com/zwave-js/node-zwave-js/commit/bca41115e5f3b7e3dcf20db6858f4d0a4b0ee3e9?diff=unified

**Comment by AlCalzone** [comment, marked as answer]
It is in a branch in your fork: https://github.com/Masteroshi430/node-zwave-js/tree/patch-1
![grafik](https://user-images.githubusercontent.com/17641229/190978678-ae429b4c-b255-4549-ba31-f6dc1699e5d3.png)
You need to actually create a pull-request using the button on the right.

**Comment by Masteroshi430** [comment]
Thanks, I just did it.
